mgmt/rest/rbody/plugin.go

```diff
@@ -107,4 +107,5 @@ type AvailablePlugin struct {
 	LastHitTimestamp int64  `json:"last_hit_timestamp"`
 	ID               uint32 `json:"id"`
 	Href             string `json:"href"`
+	PprofPort        string `json:"pprof_port"`
```

Comment:
At the end the goal is to display the port as a string when typing `snapctl plugin list --running`. Communication between snapctl and snapd are in json, which mean string anyway.